common/src/main/java/com/microsoft/identity/common/internal/platform/Algorithm.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.identity.common.internal.platform;
+
+/**
+ * Marker interface for crypto algorithm names.
+ */
+public interface Algorithm {
+    /**
+     * @return the standard algorithm name;
+     */
+    public String name();
```

Comment:
`public` here is technically optional, right?

Reply:
Yeah, it is redundant.  I'm used to my IDE generating an error for these sorts of things.